web/src/components/dashboard/SidebarCardFundingNudge.test.tsx

```diff
@@ -55,6 +55,7 @@ describe("<SidebarCardFundingNudge />", () => {
         preferences: generatePreferences({ visibleSidebarCards: [fundingNudge] }),
       });
 
+      expect(screen.getByTestId("funding-nudge")).toHaveClass("primary-gradient-reverse");
```

Comment:
pebble, same as above, I don't think this is needed, or create a new test

Reply:
deleted

Reply:
[pebble] If we want to test the gradients are present, let's check against all of the modified cards not just 2 instances inside functionality tests. Maybe make a set of related tests nested under their own describe that check the presence of each expected gradient.